addon/wolfcrypt/main.cpp

```diff
@@ -28,11 +28,22 @@
 #include "./h/ecc.h"
 #include "./h/pbkdf2.h"
 #include "./h/pkcs7.h"
+#include "./h/pkcs12.h"
 
 using namespace Napi;
 
 Napi::Object Init(Napi::Env env, Napi::Object exports)
 {
+#ifdef HAVE_FIPS
+#ifdef WC_RNG_SEED_CB
+    wc_SetSeed_Cb(wc_GenerateSeed);
+#endif
+    if (wc_RunAllCast_fips() != 0) {
```

Comment:
The DLL initializer should run the tests already. We shouldn't be calling this ourselves.

Reply:
removed, was originally part of debugging an issue but is unnecessary.